pkg/aggregator/demultiplexer.go

```diff
@@ -138,7 +138,9 @@ func sendIterableSeries(serializer serializer.MetricSerializer, start time.Time,
 // GetDogStatsDWorkerAndPipelineCount returns how many routines should be spawned
 // for the DogStatsD workers and how many DogStatsD pipeline should be running.
 func GetDogStatsDWorkerAndPipelineCount() (int, int) {
-	return getDogStatsDWorkerAndPipelineCount(agentruntime.NumVCPU())
+	work, pipe := getDogStatsDWorkerAndPipelineCount(agentruntime.NumVCPU())
+	log.Infof("Decided to run %d workers and %d pipelines", work, pipe)
```

Comment:
Maybe mention what the workers and pipelines refer to (the package name is included in the output and is a hint, but I think it may be clearer to be explicit).

```suggestion
	log.Infof("Dogstatsd configured to run with %d workers and %d pipelines", work, pipe)
```